modules/disabling-insights-operator-gather.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * support/remote_health_monitoring/using-insights-operator.adoc
+
+
+:_content-type: PROCEDURE
+[id="disabling-insights-operator-gather_{context}"]
+= Disabling the Insights Operator gather operations
+
+You can disable the Insights Operator gather operations. Disabling the gather operations gives you the ability to increase privacy for your organization as Insights Operator will no longer gather and send Insights cluster reports to Red Hat. This will disable Insights analysis and recommendations for your cluster without affecting other core functions that require communication with Red Hat such as cluster transfers. You can view a list of attempted gather operations for your cluster from the `/insights-operator/gathers.json` file in your Insights Operator archive. Be aware that some gather operations only occur when certain conditions are met and may not appear in your most recent archive. 
+
+:FeatureName: InsightsDataGather
```

Comment:
This should be in monospace and have the object type following it. I think it's a CR, but if it's another kind of object put that instead :) 

```suggestion
:FeatureName: The `InsightsDataGather` custom resource
```

Reply:
I apologize @jeana-redhat! I think I got them all now.

Reply:
no need to apologize! thanks for following up :)